CONTRIBUTING.md

```diff
@@ -0,0 +1,94 @@
+# Contributing to fzf-eip 🔍 ♢
+
+We love your input! We want to make contributing to this project as easy and transparent as possible, whether it's:
+
+- Reporting a bug
+- Discussing the current state of the code
+- Submitting a fix
+- Proposing new features
+- Becoming a maintainer
+
+## We Develop with Github
+
+We use github to host code, to track issues and feature requests, as well as accept pull requests.
+
+## Workflow
+
+We use [Github Flow](https://guides.github.com/introduction/flow/index.html).
+
+All code changes happen through pull requests. They are the best way to propose changes to the codebase.
+
+We actively welcome your pull requests:
+
+1. Fork the repo and create your branch from `master`.
+2. If you've added code that should be tested, add tests.
+3. If you've changed APIs, update the documentation.
+4. Ensure the test suite passes.
+5. Make sure your code lints.
```

Comment:
These aren't necessary. There are no code tests currently

Reply:
Agreed, Removing and adding reference to familarize with Gitflow workflow